File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.cpp
class Solution {
public:
    int maxProduct(int n) {
        vector<int> nums;

        while(n>0){
            nums.push_back(n%10);
            n /= 10;
        }
        int len = nums.size();
        sort(nums.begin(), nums.end());
        return nums[len - 1] * nums[len - 2];
    }
};

File: 3536-maximum-product-of-two-digits/3536-maximum-product-of-two-digits.py
class Solution:
    def maxProduct(self, n: int) -> int:
        nums : List[int] = []

        while(n > 0):
            nums.append(int(n % 10))
            n /= 10
        
        nums.sort()
        n = len(nums)
        return nums[n - 1] * nums[n - 2]